tests/unit_tests/test_embedding_function_registry.py

```
@@ -0,0 +1,432 @@
+                return "typed_embedding"
+
+            def __call__(self, _input: Documents) -> Embeddings:
+                return [[0.1, 0.2, 0.3]]
+
+            def get_config(self) -> Dict[str, Any]:
+                return {"model_name": self.model_name}
+
+            @staticmethod
+            def build_from_config(_config: Dict[str, Any]) -> "TypedEmbeddingFunction":
+                return TypedEmbeddingFunction(
+                    model_name=config.get("model_name", "test")
```

Ruff (0.14.13)

Undefined name `config`

(F821)